tests/integration/standard/test_control_connection.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+#
+#
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest  # noqa
+
+
+from cassandra.cluster import Cluster
+from tests.integration import use_singledc, PROTOCOL_VERSION
+from tests.integration.datatype_utils import update_datatypes
+
+
+def setup_module():
+    use_singledc()
+    update_datatypes()
+
+
+class ControlConnectionTests(unittest.TestCase):
+    def setUp(self):
+        self.cluster = Cluster(protocol_version=PROTOCOL_VERSION)
+        self.session = self.cluster.connect()
+
+    def tearDown(self):
+        self.session.execute("DROP KEYSPACE IF EXISTS keyspacetodrop ")
```

Comment:
We have typically avoided {{IF EXISTS}} because it doesn't work in C* < 2.0. Not sure if you want to maintain the ability to test there...